client/blocks/product-purchase-features/product-purchase-features-list/video-audio-posts.jsx

```diff
@@ -12,17 +12,32 @@ import { localize } from 'i18n-calypso';
  */
 import PurchaseDetail from 'components/purchase-detail';
 import paths from 'lib/paths';
+import { PLAN_BUSINESS } from 'lib/plans/constants';
+
+export default localize( ( { selectedSite, plan, translate } ) => {
+	let featureDescription;
+
+	switch ( plan ) {
+		case PLAN_BUSINESS:
+			featureDescription = translate(
+				'Enrich your posts with video and audio, uploaded directly on your site. ' +
+					'The Business plan also adds unlimited file storage.'
+			);
+			break;
+		default:
+			featureDescription = translate(
+				'Enrich your posts with video and audio, uploaded directly on your site. ' +
```

Comment:
Hi! I've found a possible matching string that has already been translated **21** times:
`translate( 'Enrich your posts with video and audio, uploaded directly on your site. No ads or limits. The Premium plan also adds 10GB of file storage.' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Note, I removed the `No ads or limits` bit because it doesn't make sense to say "No ads or limits. The Premium plan also adds 10GB of file storage" since 10GB is a limit then. I _think_ what we mean is "No ads on the video or limits on video length," which I can update. Otherwise, by itself, "No ads or limits" wasn't super clear.

Reply:
Makes sense to me but I can't comment on the wording. :) Mind pinging somebody who can?